### ISSUE TITLE
4.05-BETA interpreter fails with module issue

### ISSUE THREAD

**Issue by rvjansen** [RexxLA/NetRexx#54]
on openjdk 19.0.2 2023-01-17
https://github.com/RexxLA/rexx-repository/blob/master/ARB/standards/work-in-progress/code/test_c2x.nrx
(but also the other program in that directory)

```rexx
NetRexx portable processor 4.05-beta build 84-20230427-1233
Copyright (c) RexxLA, 2011,2023.   All rights reserved.
Parts Copyright (c) IBM Corporation, 1995,2008.
Program test_c2x.nrx

  === class dumper ===
    function dump(Rexx,Rexx,Rexx)
===== Exec: test_c2x =====
Example 1
string:     René
length:     4
native c2x: 52656EE9
characters: ===== Exception running class test_c2x: java.lang.ClassCastException: class java.lang.Character cannot be cast to class netrexx.lang.Rexx (java.lang.Character is in module java.base of loader 'bootstrap'; netrexx.lang.Rexx is in unnamed module of loader 'app') =====
     --- in dumper.dump(Rexx) [test_c2x.nrx:164]
 176 +++         rexxC = Rexx(c)
     +++                 ^^^^
     --- in test_c2x.main(String[]) [test_c2x.nrx:89]
  89 +++ dumper.dump("René")
     +++        ^^^^
Processing of 'test_c2x.nrx' complete [2 classes]
```

**Comment by rvjansen** [comment]
and for clarity: when compiled, there is no problem executing the test_c2x.class

**Comment by remesm** [comment]
Fixed with commit d5175a3ca1825fe06ecb3abf30f60471df37071b in NrLoop.nrx
